browser/views.py

```diff
@@ -687,3 +686,47 @@ def delete_investigator_record(request, person_id, research_id):
         Investigator.objects.filter(id=research_id).delete()
 
     return HttpResponseRedirect(reverse('person', args=(person_id,)))
+
+
+@staff_member_required
+def position(request, person_id, position_id=None):
+    person = get_object_or_404(Person, pk=person_id)
+    template = "browser/position.html"
+    form = PositionForm()
+    context = {
+        'form': form,
+        'person': person,
+        'position_id': None
```

Comment:
shouldn't position_id be position_id (what is passed into the method)?

Reply:
Yes, the position_id should be set to position_id. I have changed the above code to ```'position_id' : position_id```